### PR TITLE
[containers-ordered-strict]: reexported functions.

### LINKED ISSUE #24: Cleanup `containers-ordered-strict` package implementation by reexporting functions directly from `container` package
Like on this slide:

* http://slides.com/shersh/picnic#/10

### PR DESCRIPTION
**Pull request by piq9117**
reexported functions from containers, Data.Map.Strict.

Resolves #24

### CODE REVIEW

File: containers-ordered-strict/src/Map/Ord.hs [hunk 1]
```diff
@@ -4,26 +4,26 @@ module Map.Ord
...
 
-       , empty
-       , singleton
-       , fromList
-
-       , null
-       , size
-       , member
-       , lookup
-       , lookupDefault
```
Comment (ChShersh) [inline containers-ordered-strict/src/Map/Ord.hs]: This function is called `lookupDefault` in the signature, so, unfortunately, this particular function you can't export as `M.findWithDefault`. It should be written in previous explicit way.

File: containers-ordered-strict/src/Map/Ord.hs [hunk 2]
```diff
@@ -4,26 +4,26 @@ module Map.Ord
...
+       , M.fromList
+
+       , M.null
+       , M.size
+       , M.member
+       , M.lookup
+       , M.findWithDefault
+
+       , M.toList
+       , M.keys
```
Comment (ChShersh) [inline containers-ordered-strict/src/Map/Ord.hs]: `toList` and `keys` functions have different constraints in signature and in `Data.Map.Strict` function. So, again, unfortunately, you can't just export them from the qualified module 😞 These two functions should be written explicitly as before.

File: containers-ordered-strict/src/Map/Ord.hs [hunk 3]
```diff
@@ -4,26 +4,26 @@ module Map.Ord
...
+
+       , M.insert
+       , M.insertWith
+       , M.adjust
+       , M.update
+       , M.delete
+       , M.alter
        ) where
 
 import Prelude hiding (lookup, null)
```
Comment (ChShersh) [inline containers-ordered-strict/src/Map/Ord.hs]: I think this `Prelude` import is not required anymore.

**Comment by piq9117** [comment]
No problem! I’ll correct those. Thanks for the guidance! 😃

**Pull request closed without merging**